public/js/test/node-unit-tests.js

```diff
@@ -23,18 +22,19 @@ const webpack = require("webpack");
 const webpackConfig = require("../../../webpack.config");
 delete webpackConfig.entry.bundle;
 
+// The source map worker is compiled with webpack (and mock-require
+// doesn't work in workers) so mock it with an alias, and tweak a few
+// things to make the stub fetcher work in node.
+webpackConfig.resolve.alias.networkRequest =
+  path.join(__dirname, "/stubNetworkRequest.js");
+webpackConfig.externals = [{ fs: "commonjs fs" }];
+webpackConfig.node = { __dirname: false };
```

Comment:
It tells webpack not to mess with `__dirname`. Otherwise, `__dirname` is `/` in the modules, but because we are actually running in node, we want the real `__dirname`, and this lets us use it instead of webpack's overriden value.